mod_ci/controllers.py

```diff
@@ -460,7 +460,38 @@ def progress_reporter(test_id, token):
                 repository = gh.repos(g.github['repository_owner'])(
                     g.github['repository'])
                 # If status is complete, remove the Kvm entry
+                u1 = GeneralData.query.filter(
```

Comment:
You moved the last_running_test_id_all down, why didn't you move these two down into the if too?